dotCMS/src/main/java/com/dotcms/contenttype/business/FieldAPIImpl.java

```diff
@@ -246,6 +253,42 @@ public Field save(final Field field, final User user, final boolean reorder)
 
         return result;
     }
+    /**
+     *  Checks if the CleanUpFieldReferencesJob is running for a field with the same name, if so, throws an exception
+     * */
+    public void validateFieldReferences(Field field) throws DotDataException{
+        try {
+
+            List<JobExecutionContext> currentlyExecutingJobs = new ArrayList<>();
```

Comment:
This should be in another place, hard dependency here. May be a method on QuartzUtils